### DISCUSSION TITLE
为什么设置mysql网站就打不开了

### DISCUSSION THREAD

**Discussion by Sodiseng007** [AlistGo/alist#2659] [General]
[GIN] 2022/12/09 - 18:23:48 | 200 |  2.991555901s |   36.32.156.227 | POST     "/api/fs/get"
[GIN] 2022/12/09 - 18:23:50 | 200 |  1.822837068s |   36.32.156.227 | POST     "/api/fs/list"
[GIN] 2022/12/09 - 18:24:26 | 200 |   2.12707963s |   36.32.156.227 | POST     "/api/fs/get"
[GIN] 2022/12/09 - 18:25:12 | 200 |     569.158µs |  223.198.89.242 | GET      "/"
[GIN] 2022/12/09 - 18:25:14 | 200 |    2.063722ms |  223.198.89.242 | GET      "/api/public/settings"
[GIN] 2022/12/09 - 18:25:14 | 200 |    1.618516ms |  223.198.89.242 | GET      "/api/me"
[GIN] 2022/12/09 - 18:25:15 | 200 |    1.663871ms |  223.198.89.242 | POST     "/api/fs/get"
[GIN] 2022/12/09 - 18:25:15 | 200 |    1.076968ms |  223.198.89.242 | POST     "/api/fs/list"
[GIN] 2022/12/09 - 18:27:19 | 200 |     488.597µs |  223.198.89.242 | GET      "/"
[GIN] 2022/12/09 - 18:27:21 | 200 |     183.283µs |  223.198.89.242 | GET      "/"
[GIN] 2022/12/09 - 18:27:22 | 200 |     593.204µs |  223.198.89.242 | GET      "/assets/index.659f4289.css"
[GIN] 2022/12/09 - 18:27:22 | 200 |    1.912051ms |  223.198.89.242 | GET      "/assets/index.3a80bc76.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |     242.706µs |  223.198.89.242 | GET      "/assets/entry.56788660.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |    1.847667ms |  223.198.89.242 | GET      "/api/public/settings"
[GIN] 2022/12/09 - 18:27:22 | 200 |    1.395918ms |  223.198.89.242 | GET      "/api/me"
[GIN] 2022/12/09 - 18:27:22 | 200 |     110.412µs |  223.198.89.242 | GET      "/assets/useTitle.27a08ba5.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |     299.983µs |  223.198.89.242 | GET      "/assets/Layout.6526da94.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |      66.893µs |  223.198.89.242 | GET      "/assets/useT.f153f830.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |      59.336µs |  223.198.89.242 | GET      "/assets/api.7f233fde.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |     906.975µs |  223.198.89.242 | GET      "/assets/Markdown.ab2170d8.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |     180.221µs |  223.198.89.242 | GET      "/assets/index.68d10b5a.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |     114.042µs |  223.198.89.242 | GET      "/assets/useUtil.a5be35bc.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |      37.301µs |  223.198.89.242 | GET      "/assets/index.6289ff4c.js"
[GIN] 2022/12/09 - 18:27:22 | 200 |      56.608µs |  223.198.89.242 | GET      "/assets/FolderTree.5443c115.js"
[GIN] 2022/12/09 - 18:27:23 | 200 |    1.580949ms |  223.198.89.242 | POST     "/api/fs/get"
[GIN] 2022/12/09 - 18:27:40 | 200 |     1.02537ms |  223.198.89.242 | GET      "/api/public/settings"
[GIN] 2022/12/09 - 18:27:41 | 200 |      794.28µs |  223.198.89.242 | GET      "/api/me"
[GIN] 2022/12/09 - 18:28:14 | 200 |     1.64004ms |  223.198.89.242 | POST     "/api/fs/get"
[GIN] 2022/12/09 - 18:28:15 | 200 |    1.223308ms |  223.198.89.242 | POST     "/api/fs/list"
[GIN] 2022/12/09 - 18:28:28 | 200 |     219.982µs |  47.103.121.108 | GET      "/"
[31mFATA[0m[2022-12-09 18:29:20] failed to connect database:Error 1045: Access denied for user 'xiaokonglong'@'ecs-mlPOz' (using password: YES)

**Comment by Sodiseng007** [comment]
![image](https://user-images.githubusercontent.com/84292987/206682448-c267ef78-a530-474c-b1e7-102a68327e1c.png)

**Comment by anwen-anyi** [comment]
Access denied for user 'xiaokonglong'@'ecs-mlPOz' (using password: YES)

日志里提示你了吧？emmmmm 密码不对吗
